File: Imperium/src/com/imperium/Commands/RebootCommand.java
package com.imperium.Commands;

public class RebootCommand implements ICommand {
    private CommandType type;
    private int length;

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public RebootCommand(){
        type = CommandType.Reboot;
    }
}

File: Responder/Responder/Commands/RebootCommand.cs
namespace Responder.AppConfig
{
    public class RebootCommand : ICommand
    {
        #region Properties
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        #endregion

        #region Constructors
        public RebootCommand()
        {
            Type = CommandType.Reboot;
        }
        #endregion
    }
}
